rclcpp/cmake/rclcpp_create_node_main.cmake

```diff
@@ -0,0 +1,28 @@
+# Copyright 2015 Open Source Robotics Foundation, Inc.
```

Comment:
Sorry, the content was moved from the previous into a separate file (to match almost all other CMake functions). I don't us that CMake function anywhere.